### PR TITLE
Bump to Xcode 9

### PR DESCRIPTION
**Pull request by VincentDondain**
Update Versions-ios and Versions-mac file too.

### CODE REVIEW

File: Make.config [hunk 1]
```diff
@@ -40,17 +40,17 @@ IOS_PRODUCT=Xamarin.iOS
...
 IOS_PACKAGE_VERSION_BUILD=$(IOS_COMMIT_DISTANCE)
 IOS_PACKAGE_UPDATE_ID=$(shell printf "2%02d%02d%02d%03d" $(IOS_PACKAGE_VERSION_MAJOR) $(IOS_PACKAGE_VERSION_MINOR) $(IOS_PACKAGE_VERSION_REV) $(IOS_PACKAGE_VERSION_BUILD))
 
-# Xcode 8.3
-XCODE_VERSION=8.3
-XCODE_URL=http://xamarin-storage/bot-provisioning/Xcode_8.3.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode83.app/Contents/Developer
+# Xcode 9
+XCODE_VERSION=9
+XCODE_URL=http://xamarin-storage/bot-provisioning/Xcode_9.xip
```
Comment (dalexsoto) [inline Make.config]: I think you meant http://xamarin-storage/bot-provisioning/Xcode_9_beta.xip

Comment (VincentDondain) [inline Make.config]: Just assumed the wrong link (:

**Comment by spouliot** [comment]
Please review failures and post them on slack, if help is needed, until the branch builds :)

**Comment by VincentDondain** [comment]
Don't merge, I will do a force push at the end to refactor the commits (I don't think we should just squash them this time).

Will do this once it's a. building and b. approved.

**Comment by VincentDondain** [comment]
After 3 hr 12 min the build seems stuck but we can build locally with these fixes.

Slack conversation:

```
Sebastien Pouliot [16:13] 
If it's running tests then it has completed building (good)
Also I don't think Rolf 's fix for the sim are part of the pr

Alex Soto [16:13] 
So might be hung right know on sims

Sebastien Pouliot [16:14] 
So even if not stuck it will all fail

Alex Soto [16:14] 
should we #YoloMerge ?

Sebastien Pouliot [16:14] 
It might only be consecutive timeouts
Yes, let's merge that one so everyone can build it locally

Vincent Dondain [16:15] 
ok doing it
```

So let's squash merge this so people can indeed build (:

**Pull request merged**